devtron-labs/dashboard
# Fix - Multiple container headers & added header note for external link discovery

# Description

These changes address the following,

1. AppDetails - Multiple container headers are showing in K8s resources after expanding node details.
2. AppDetails - Added header note for the discovery of container-level external links 
3. Export to CSV - minor text change based on processing state

Resolves https://github.com/devtron-labs/devtron/issues/2242 and resolves https://github.com/devtron-labs/devtron/issues/2243

## Type of change


# How Has This Been Tested?

All tests have been performed manually by,
- Visit the AppDetails page > scroll down to node details > expand node details > observe > there should be only one header or multiple distinct headers.
- Create a container level external link > Visit the AppDetails page >  scroll down to node details > observe > there should be a header note saying -  `Expand pods to view external links for containers.`
- Visit the AppListing page > click on export to CSV > observe > It should show the cancel button while processing & close button after successfully exporting the data.


# Checklist:
